exercises/concept/wizards-and-warriors/.docs/hints.md

```diff
@@ -2,41 +2,51 @@
 
 ## General
 
-Detailed explanation of inheritance can be found at [Inheritance][inheritance-main].
+Detailed explanation of inheritance can be found at [Inheritance][inheritance-concept].
 The whole inheritance concept has a lot to do with the concepts around [overriding][java-overriding].
 
-## 1. Describe a Fighter
+## 1. Create the Warrior and Wizard classes
+
+- Review the [class][class-concept] concept.
```

Comment:
This might be a good place to use [widgets](https://exercism.org/docs/building/markdown/widgets) to link to the concept on the website

Reply:
Done!

Reply:
Seems like the linter check does not like the widgets or I'm doing something wrong

Reply:
> The link reference, i.e. the contents of (), is ignored by the website. You can point them at whatever you want for maintenance purposes.

Oh yeah, It's because it should point to somewhere, where do you think I should point it ?

Reply:
I'd say point it to the source on GitHub, so https://github.com/exercism/java/tree/main/concepts/classes